data_steward/cdr_cleaner/manual_cleaning_rules/dc732_suppress.py

```diff
@@ -0,0 +1,445 @@
+# # DC-732 Retract Rows with Suppressed Concepts
+#
+# Identify, log and remove all rows from all tables in specified deid datasets where at least one concept_id field references a concept which must be suppressed.
+#
+# ## Notes
+#  * Prior to running the table identified by `CONCEPT_LOOKUP_TABLE` must be loaded with concepts to suppress. At minimum this table should have the field `concept_id`.
+#  * This can safely be re-run (e.g. if a new concept is added to the lookup).
+# <style>
+# /* make tables look less bad in jupyter */
+# table {margin-left: 0 !important;}
+# </style>
+# +
+from google.cloud import bigquery
+import jinja2
+import pandas as pd
+
+
+from utils import bq
+# -
+
+ISSUE_NUMBER = 'DC-732'
+ISSUE_PREFIX = 'dc732_'
+ROWS_RESOURCE_NAME = f'{ISSUE_PREFIX}suppress_rows'
+SUMMARY_RESOURCE_NAME = f'{ISSUE_PREFIX}suppress_summary'
+
+JINJA_ENV = jinja2.Environment(
+    # block tags on their own lines
+    # will not cause extra white space
+    trim_blocks=True,
+    lstrip_blocks=True,
+    # syntax highlighting should be better
+    # with these comment delimiters
+    comment_start_string='--',
+    comment_end_string=' --')
+CLIENT = bq.get_client()
```

Comment:
I would prefer if we provided the project_id either via a config file or a command line parameter.  I know this works right now, but I think it's very susceptible to someone forgetting to reset their environment variables and possibly wiping out data that we don't want gone.

Reply:
You're right, as ad hoc as this is, I shouldn't make that kind of error so easy to commit; it'd be like setting a trap for myself. I updated this so these parameters now come from command line args.

Reply:
I fell in that trap myself with testing.  Thankfully, I had a check that failed because my environment didn't have 'test' in the name.  But yea, I forgot to switch from prod configuration to test/dev configuration.  So, hooray for failures!  I would love to make a way for this to be harder to forget by using either command line args or config files.